src/utils/returns.ts

```diff
@@ -266,6 +265,55 @@ export async function getLPReturnsOnPair(client, user: string, pair, nativeCurre
     return entry.pair.id === pair.id;
   });
 
+  // snaphots to be used for the fees computation
+  const derivedSnapshots = mergeLiquiditySnapshot(snapshots);
+
+  // get data about the current position
+  const derivedCurrentPosition: Position = {
+    pair,
+    liquidityTokenBalance: derivedSnapshots[derivedSnapshots.length - 1]?.liquidityTokenBalance,
+    liquidityTokenTotalSupply: pair.totalSupply,
+    reserve0: pair.reserve0,
+    reserve1: pair.reserve1,
+    reserveUSD: pair.reserveUSD,
+    token0PriceUSD: pair.token0.derivedNativeCurrency * nativeCurrencyPrice,
+    token1PriceUSD: pair.token1.derivedNativeCurrency * nativeCurrencyPrice,
+  };
+
+  for (const index in derivedSnapshots) {
+    // get positions at both bounds of the window
+    const positionT0 = derivedSnapshots[index];
+    const positionT1 =
+      parseInt(index) === derivedSnapshots.length - 1 ? derivedCurrentPosition : derivedSnapshots[parseInt(index) + 1];
+
+    const results = getMetricsForPositionWindow(positionT0, positionT1);
+    netReturn += results.netReturn;
+    swaprReturn += results.swaprReturn;
+    fees += results.fees;
```

Comment:
```suggestion
    netReturn = netReturn + results.netReturn;
    swaprReturn = swaprReturn + results.swaprReturn;
    fees = fees + results.fees;
```
I personally prefer not use shot-hands for easy code reading. Not a biggie. If you feel its easy for you keep it. Since we compile and uglify code these all will be optimized.